app/src/main/res/values/component_colors.xml

```diff
@@ -160,4 +160,11 @@
   <color name="component_color_math_expression_parser_activity_radio_buttons_color">@color/color_palette_dark_text_color</color>
   <color name="component_color_math_expression_parser_activity_edit_text_color">@color/color_palette_dark_text_color</color>
   <color name="component_color_math_expression_parser_activity_hint_text_color">@color/color_palette_hint_text_color</color>
+  <!-- Audio Language Activity -->
```

Comment:
Please add similar comments for reading text size and app language related actvities.

Reply:
Actually I added that at line 168.